.github/workflows/sync_docs.yaml

```diff
@@ -4,16 +4,17 @@ on:
   workflow_dispatch:
   schedule:
     - cron: '53 0 * * *' # Daily at 00:53 UTC
+  # Triggered on push to branch "main" by .github/workflows/release.yaml
+  workflow_call:
 
 jobs:
-
   sync-docs:
     name: Open PR with docs changes
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
       - name: Open PR with docs changes
-        uses: canonical/upload-charm-docs@main
+        uses: deusebio/discourse-gatekeeper@c8adb89ea1cbceca54d78da798658373615487ac
```

Comment:
Should/will be replaced with https://github.com/canonical/discourse-gatekeeper once published there.

Reply:
What's keeping it from moving to canonical org?

Reply:
[This](https://github.com/deusebio/discourse-gatekeeper/commit/4f7532da97a88dca5dc244e0dc1d1117a23c37da) patch. Ideally, this is not needed if:
1. Two-ways sync is enabled (which however we can't scale more broadly until we got upper leadership ok)
2. [This] spec is finally implemented (which would structure sync between Navigation Table and contents Index robust and reproducible

Maintainer of Discourse Gatekeeper did not want to merge the patch, since it will hopefully be solved once (2) is finalized. But for the time being, if one-way sync wants to be applied to our repo, this is the way to go I suppose. :(